pages/_app.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import NextApp from 'next/app'
-
```

Comment:
I would keep it to explicitly divide global and local but this is up to you.

Reply:
I agree. Please check this [PR](https://github.com/oacore/configs/pull/11). After we merge it I will do proper changes here. Unfortunately, this rule doesn't apply on assets imports. I propose we should have .css/.svg import as a separate 4th group.